text/0858-deprecate-ember-mocha.md

```diff
@@ -0,0 +1,80 @@
+---
+stage: accepted
+start-date: 2022-11-08
+release-date:
+release-versions:
+teams: # delete teams that aren't relevant
+  - data
+  - framework
+  - learning
+prs:
+  accepted: https://github.com/emberjs/rfcs/pull/858
+project-link:
+---
+
+<!---
+Directions for above:
+
+stage: Leave as is
+start-date: Fill in with today's date, 2032-12-01T00:00:00.000Z
+release-date: Leave as is
+release-versions: Leave as is
+teams: Include only the [team(s)](README.md#relevant-teams) for which this RFC applies
+prs:
+  accepted: Fill this in with the URL for the Proposal RFC PR
+project-link: Leave as is
+-->
+
+# Deprecate `ember-mocha`
+
+## Summary
+
+This RFC proposes to officially deprecate [`ember-mocha`](https://github.com/emberjs/ember-mocha).
+
+Consequently, this would also deprecate support for `ember-mocha` when 
+generating test blueprints. Both `ember-source` and `ember-data` have logic to 
+determine the presence of `ember-mocha` in order to use the appropriate test blueprints.
+
+## Motivation
+
+`ember-mocha` has been unmaintained for a while now. The last release was 
+published on [Jun 16, 2019](https://github.com/emberjs/ember-mocha/releases/tag/v0.16.0). 
+It also seems this release is [not compatible with Ember v4](https://github.com/emberjs/ember-mocha/issues/691). 
+Instead of letting `ember-mocha` slowly fade away, it feels better to officially deprecate 
+it and clearly communicate this to the community.
+
+## Transition Path
+
+We should:
+
+- Update `ember-mocha`'s README to state that `ember-mocha` is officially 
+deprecated, and that users should consider migrating to [`ember-qunit`](https://github.com/emberjs/ember-qunit) instead
+- Officially mark `ember-mocha` as deprecated on [the npm registry](https://www.npmjs.com/)
+- Update the test-framework detectors in `ember-source` and `ember-data` to 
+deprecate support for `ember-mocha`:
+  - https://github.com/emberjs/ember.js/blob/master/blueprints/test-framework-detector.js
+  - https://github.com/emberjs/data/blob/master/packages/private-build-infra/src/utilities/test-framework-detector.js
+- Write a small guide that explains how to migrate from `ember-mocha` to `ember-qunit`
+
+## How We Teach This
+
+References to `ember-mocha` should be removed from all learning materials, for example:
+
+- https://guides.emberjs.com/release/testing/testing-tools/#toc_mocha-chai-dom
+- https://guides.emberjs.com/release/testing/testing-tools/#toc_summary
+- https://guides.emberjs.com/release/testing/#toc_how-to-filter-tests
+
+## Drawbacks
+
+People using `ember-mocha` will have to migrate to using `ember-qunit` instead 
```

Comment:
They'll have to migrate anyway unless someone starts maintaining it, and it doesn't sound like people are leaping at that chance.

Reply:
This is great, thanks for sharing @octo-topi!

Reply:
@octo-topi I was amazed by the quality of your pull request. Texts can be auto-translated, no worries.

I think the one-to-one mapping between assertions in `mocha` and `qunit`, shown in [commit 3](https://github.com/1024pix/pix/pull/5258/commits/b0eccbdb63caee853d67d2b368c9f6079a334a08), will help other teams with their migration. 💯

Reply:
Well, @VincentHardouin is actually the one who has the job done, helped by several other members.
As we have an open-source repository, it was worth share the result so other people can focus on less tedious task.